css/types/counters.json

```diff
@@ -0,0 +1,58 @@
+{
+  "css": {
+    "types": {
+      "attr": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/counters",
+          "description": "<code>counters()</code>",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": "12"
+            },
+            "edge_mobile": {
+              "version_added": "12"
+            },
+            "firefox": {
+              "version_added": "1.5"
+            },
+            "firefox_android": {
+              "version_added": "31"
```

Comment:
Seems a little odd for this to be "31" since Firefox Desktop implemented support in 1.5 based upon your data.  Do you know why this may be?

Reply:
Thanks for the change!  “4” is the first Firefox Android version that was released, which is why BCD isn’t letting you pick anything prior to it.  Looks good to me now!